### DISCUSSION TITLE
TabsContent animate

### DISCUSSION THREAD

**Discussion by brokuka** [unovue/shadcn-vue#692] [Q&A] [answered]
Is there way to animate tabs content?

**Comment by sadeghbarati** [comment, marked as answer]
Yes you can by using 

- Vue Transition component
- [`@vueuse/motion` ](https://motion.vueuse.org/)
- [`motion.dev`](https://motion.dev/)
- [`tailwindcss-animate`](https://github.com/jamiebuilds/tailwindcss-animate)
- etc xd ..


https://stackblitz.com/edit/weussu?file=src%2Fmain.ts,src%2FApp.vue (this example is with `@vueuse/motion`)
